core/src/main/java/io/camunda/migrator/impl/persistence/IdKeyDbModel.java

```diff
@@ -7,16 +7,19 @@
  */
 package io.camunda.migrator.impl.persistence;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
 import io.camunda.migrator.impl.persistence.IdKeyMapper.TYPE;
 import java.util.Date;
 import java.util.Objects;
 
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
```

Comment:
This is required for executing queries from the cockpit plugin

Reply:
/q I assume the test you wrote fails when this annotation is removed, right?

Reply:
The test was passing, the count queries worked but fetching returned empty results. 
I pushed changes to fix this. Renamed getters to follow JavaBean conventions so they are recognised during deserialisation.